python/triton/language/libdevice.py

```diff
@@ -975,10 +1000,16 @@ def rsqrt_rn(arg0, _builder=None):
 
 @extern.extern
 def ffs(arg0, _builder=None):
-    return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, ],
-                              {(core.dtype("int32"),): ("__nv_ffs", core.dtype("int32")),
-                               (core.dtype("int64"),): ("__nv_ffsll", core.dtype("int32")),
-                               }, _builder)
+    if torch.version.hip is not None:
+        return extern.elementwise("libdevice", LIBDEVICE_PATH, [arg0, ],
```

Comment:
This is wrong so the corresponding unit test in test_ext_elemwise has run time errors with 67% mismatched elements.

So this is still a WIP.